addons/binding/pom.xml

```diff
@@ -24,6 +24,7 @@
     <module>org.openhab.binding.feed</module>
     <module>org.openhab.binding.feed.test</module>
     <module>org.openhab.binding.freebox</module>
+    <module>org.openhab.binding.gardena</module>
```

Comment:
please also add it to https://github.com/openhab/openhab2-addons/blob/master/features/openhab-addons/src/main/feature/feature.xml